pkg/rpcclient/nep22/nep22.go

```diff
@@ -0,0 +1,70 @@
+package nep22
+
+import (
+	"github.com/nspcc-dev/neo-go/pkg/core/transaction"
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract"
+	"github.com/nspcc-dev/neo-go/pkg/util"
+)
+
+// Actor is used by Token to create and send transactions.
+type Actor interface {
+	MakeRun(script []byte) (*transaction.Transaction, error)
+	MakeUnsignedRun(script []byte, attrs []transaction.Attribute) (*transaction.Transaction, error)
+	SendRun(script []byte) (util.Uint256, uint32, error)
+}
+
+// Token provides full NEP-22 interface.
+type Token struct {
+	hash  util.Uint160
+	actor Actor
+}
+
+// UpdateParameters is a set of parameters for `transfer` method.
+type UpdateParameters struct {
+	NefFile  []byte
+	Manifest []byte
+	Data     any
+}
+
+// New creates an instance of Token for contract with the given hash
+// using the given Actor.
+func New(actor Actor, hash util.Uint160) *Token {
+	return &Token{
+		hash:  hash,
+		actor: actor,
+	}
+}
+
+// Update calls the contract's "update" method in a single transaction.
+func (t *Token) Update(nef []byte, manifestBytes []byte, data any) (util.Uint256, uint32, error) {
```

Comment:
Documentation text of such methods is relatively standardized, try to follow this standard. As an example, take a look at autogenerated RPC bindings:
https://github.com/nspcc-dev/neo-go/blob/2470ebf1344a6c2b24502a251bfa076bad25a9df/pkg/smartcontract/rpcbinding/binding.go#L69-L72
https://github.com/nspcc-dev/neo-go/blob/2470ebf1344a6c2b24502a251bfa076bad25a9df/cli/smartcontract/testdata/rpcbindings/nft-d/rpcbindings.out#L66-L69

Some additional explanations may be added, but template text should not be removed or changed. Take a look at NEP17 token wrapper that contains some adjustments comparing to the auto-generated code:
https://github.com/nspcc-dev/neo-go/blob/2470ebf1344a6c2b24502a251bfa076bad25a9df/pkg/rpcclient/nep17/nep17.go#L84-L88